internal/model/utils.go

```diff
@@ -0,0 +1,15 @@
+package model
+
+import "kube-proxless/internal/utils"
+
+func compareRoute(r1, r2 *Route) bool {
```

Comment:
action = isRouteSame or areRouteEqual. compareRound doesn't sound like it would return a bool.